### PR TITLE
ref(metrics): Remove extrapolation code

### LINKED ISSUE #76782: Remove backend codebase for metrics extrapolation

### PR DESCRIPTION
**Pull request by priscilawebdev**
closes https://github.com/getsentry/sentry/issues/76782

### CODE REVIEW

**Comment by shellmayr** [comment]
@priscilawebdev something still seems to be off in the derefed-json openapi test

**Review by vgrozdanic — APPROVED** [review]
Looks good to me, I would just suggest that we wait until Monday before merging and deploying this, since there are (although straightforward) changes to project configs generation, which can break a lot of things

**Comment by priscilawebdev** [comment]
@vgrozdanic sure I am not merging my PRs today 😄

**Review by shellmayr — APPROVED** [review]
LGTM

File: src/sentry/options/defaults.py [hunk 1]
```diff
@@ -1470,36 +1470,6 @@
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
 
-# Enables extrapolation on the `transactions` namespace.
-register(
```
Comment (jan-auer) [inline src/sentry/options/defaults.py:1474]: Either remove these in a separate PR, or ensure that https://github.com/getsentry/sentry-options-automator/pull/2199 is merged _and applied_ before this PR.

**Review by jan-auer — APPROVED** [review]

**Comment by priscilawebdev** [comment]
@shellmayr Yes, I noticed that. I tried running the command to update the snapshots, but it appears to be deprecated and didn't work. In any case, the test isn't required

**Pull request merged**